test/index.js

```diff
@@ -1106,6 +1106,10 @@ describe('list', function() {
       assert(S.at(-1, ['foo', 'bar', 'baz']).equals(S.Just('baz')));
     });
 
+    it('returns Nothing when passed the index -0', function() {
+      assert(S.at(-0, ['foo', 'bar', 'baz']).equals(S.Nothing()));
+    });
```

Comment:
I suggest we move this assertion to the following block, since `-0` is an out-of-bounds index.